$MirOS: ports/emulators/qemu/patches/patch-target-i386_translate_c,v 1.1 2005/05/03 19:30:05 tg Exp $
--- target-i386/translate.c.orig	Sun Sep  4 17:11:09 2005
+++ target-i386/translate.c	Tue Nov 15 11:37:45 2005
@@ -4637,7 +4637,7 @@ static target_ulong disas_insn(DisasCont
             case 0x18 ... 0x1b:
                 {
                     int op1;
-                    const static uint8_t fcmov_cc[8] = {
+                    static const uint8_t fcmov_cc[8] = {
                         (JCC_B << 1),
                         (JCC_Z << 1),
                         (JCC_BE << 1),
